engine/userguide/storagedriver/imagesandcontainers.md

```diff
@@ -12,29 +12,51 @@ stores images. Then, you need an understanding of how these images are used by
 containers. Finally, you'll need a short introduction to the technologies that
 enable both images and container operations.
 
-## Images and layers
+Understanding how Docker manages the data within your images and containers will
+help you understand the best way to design your containers and Dockerize your
+applications, and avoid performance problems along the way.
 
-Each Docker image references a list of read-only layers that represent
-filesystem differences. Layers are stacked on top of each other to form a base
-for a container's root filesystem. The diagram below shows the Ubuntu 15.04
-image comprising 4 stacked image layers.
+## Images and layers
 
-![](images/image-layers.jpg)
+A Docker image is built up from a series of layers. Each layer represents an
+instruction in the image's Dockerfile. Each layer except the very last one is
+read-only. Consider the following Dockerfile:
+
+```conf
+FROM ubuntu:15.10
+COPY . /app
+RUN make /app
+CMD python /app/app.py
+```
+
+This Dockerfile contains four commands, each of which creates a layer.  The
+`FROM` statement starts out by creating a layer from the `ubuntu:15.10` image.
+The `COPY` command adds some files from your Docker client's current directory.
+The `RUN` command builds your application using the `make` command. Finally,
+the last layer specifies what command to run within the container.
+
+Each layer is only a set of differences from the layer before it. The layers are
+stacked on top of each other. When you create a new container, you add a new
+writable layer on top of the underlying layers. This layer is often called the
+"container layer". All changes made to the running container, such as writing
+new files, modifying existing files, and deleting files, are written to this thin
+writable container layer. The diagram below shows a container based on the Ubuntu
+15.04 image.
 
-The Docker storage driver is responsible for stacking these layers and
-providing a single unified view.
+The diagram below shows a container with the Ubuntu 15.04 image comprising 4
+stacked image layers, and the writable container layer on top.
 
-When you create a new container, you add a new, thin, writable layer on top of
-the underlying stack. This layer is often called the "container layer". All
-changes made to the running container - such as writing new files, modifying
-existing files, and deleting files - are written to this thin writable
-container layer. The diagram below shows a container based on the Ubuntu 15.04
-image.
+![Docker image layers](images/container-layers.jpg)
 
-![](images/container-layers.jpg)
+A _storage driver_ handles the details about the way these layers interact with
+each other. Different storage drivers are available, which have advantages
+and disadvantages in different situations.
 
 ### Content addressable storage
 
+If you are already running Docker 1.10 or higher, you can skip this section and
```

Comment:
@mstanleyjones 1.10 is no longer supported. Can we remove this or archive for clarity?